src/gui/qgsattributeform.h

```diff
@@ -46,18 +46,6 @@ class GUI_EXPORT QgsAttributeForm : public QWidget
 
   public:
 
-    //! Form modes
```

Comment:
To avoid an API break, this needs to stay around until QGIS 4 (but add `\deprecated` to the docs)